package.json

```diff
@@ -30,7 +30,7 @@
   },
   "devDependencies": {
     "chai": "^4.2.0",
-    "eslint": "5.16.0",
+    "eslint": "6.5.1",
```

Comment:
can you make sure to also update the `yarn.lock` file so that it does not go out-of-sync?

while you're at it, could you change this into a `^` constraint. I don't remember why this is was not using it already, but I don't see any reason why we shouldn't 🤔

Reply:
Ah, yes, yarn, entschuldigung... :)

Done, and also the `^` addition.